code/js/popup.js

```diff
@@ -59,28 +61,51 @@ function dumpBookmarks(query)
 		{
 			var current_found = 0;
 
+
 			for(var pos in single_query)
 			{
+
 				current_query = single_query[pos];
 				if(current_query == "")
 					continue;
 
 			    if ( current_found == 0 && ((String(key).toLowerCase()).indexOf(current_query.toLowerCase()) != -1)) 
 			    {
+
 				    found = 1;
 				    current_found = 1;
+
+				    let str = key;
+				    let inside_text = '';
+				    str = str.split("/").pop();
+				    str = str.split('_').join(' ');
+					str = str.replace(/\w\S*/g, function(txt) {
+						return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+					});
+				   
 				    if(obj[key].length ==1 && ((String(obj[key]).toLowerCase()).indexOf("README.md".toLowerCase()) != -1))
 				    	continue;
 				    else
-				    {
+				    {					   
+					    let s_no = 1;
```

Comment:
This variable name (`s_no`) should be more explanatory such as `sub_result_number`.